Apps/WebClient/src/Server/Services/UserPhoneService.cs

```diff
@@ -0,0 +1,79 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Services
+{
+    using System;
+    using HealthGateway.Common.Constants;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Common.Services;
+    using HealthGateway.Database.Constants;
+    using HealthGateway.Database.Delegates;
+    using HealthGateway.Database.Models;
+    using HealthGateway.Database.Wrapper;
+    using Microsoft.Extensions.Logging;
+
+    /// <inheritdoc />
+    public class UserPhoneService : IUserPhoneService
+    {
+        private readonly ILogger logger;
+        private readonly IProfileDelegate profileDelegate;
+        private readonly INotificationSettingsService notificationSettingsService;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UserEmailService"/> class.
+        /// </summary>
+        /// <param name="logger">Injected Logger Provider.</param>
+        /// <param name="profileDelegate">The profile delegate to interact with the DB.</param>
+        /// <param name="notificationSettingsService">Notification settings delegate.</param>
+        public UserPhoneService(ILogger<UserEmailService> logger,
+            IProfileDelegate profileDelegate,
+            INotificationSettingsService notificationSettingsService)
+        {
+            this.logger = logger;
+            this.profileDelegate = profileDelegate;
+            this.notificationSettingsService = notificationSettingsService;
+        }
+
+        /// <inheritdoc />
+        public bool UpdateUserPhone(string hdid, string phone, Uri hostUri, string bearerToken)
+        {
+            this.logger.LogTrace($"Updating user phone numbner...");
+            UserProfile userProfile = this.profileDelegate.GetUserProfile(hdid).Payload;
+            userProfile.PhoneNumber = phone;
```

Comment:
is phone stripped down to its core digits?  (555) 123-3456 should be stored in DB as 5551233456

Reply:
I would guess that happens on the front end model. Will check with sam since thats only representation

Reply:
Sounds good as I created a bug to update the DB model to be a max of 10 chars.